abs.yaml

```diff
@@ -0,0 +1 @@
+pbp_autopublish: false
```

Comment:
What does this do?

Reply:
It's just a temporary test for the new build platform. Right now all packages are automatically getting uploaded to pbp, this disables that to allow for a review and released with a click of the mouse. Happy to chat more offline if you'd like!